ccd-definition/ComplexTypes/CareSupervision/DocumentParty/DocumentSentToParty.json

```diff
@@ -39,5 +39,13 @@
     "FieldType": "Text",
     "ElementLabel": "ID from Send Letter Service",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DocumentSentToParty",
+    "ListElementCode": "removalReason",
```

Comment:
have we thought about what this will default to for documents that have already been removed?

Reply:
Probably, fine - guess it will just be empty